common/graph_db_client.py

```diff
@@ -42,56 +43,76 @@ def result_to_df(self,result) -> pd.DataFrame:
         df = pd.DataFrame(json_data)
         return df
 
-    def query_vertices(self) -> pd.DataFrame:
+    def query_vertices(self,context_id:str="00000000-0000-0000-0000-000000000000") -> pd.DataFrame:
         result = self._client.submit(
             message=(
                 "g.V()"
             ),
         )
         return self.result_to_df(result)
 
-    def query_edges(self) -> pd.DataFrame:
+    def query_edges(self,context_id:str="00000000-0000-0000-0000-000000000000") -> pd.DataFrame:
         result = self._client.submit(
             message=(
                 "g.E()"
             ),
         )
         return self.result_to_df(result)
 
-    def write_vertices(self,data: pd.DataFrame)->None:
-        for row in data.itertuples():
-            print(row.id)
```

Comment:
is it intentional?

Reply:
This seems to be a result of git formatting the differences in a weird way. This function was not removed, just modified the parameters of the function